src/linux/helpers/loader.c

```diff
@@ -334,14 +335,15 @@ frida_receive_fd (int sockfd, const FridaLibcApi * libc)
     .iov_len = sizeof (dummy)
   };
   FridaControlMessage control;
-  struct msghdr msg = {
-    .msg_name = NULL,
-    .msg_namelen = 0,
-    .msg_iov = &io,
-    .msg_iovlen = 1,
-    .msg_control = &control,
-    .msg_controllen = sizeof (control),
-  };
+  struct msghdr msg;
+
+  /* avoid inline initialization to prevent the compiler attempting to insert a call to  memset */
```

Comment:
Comments should start with the first word capitalized and end in a full stop.

Reply:
Done